Data/SliceHandlerItkWrapper.cpp

```diff
@@ -133,6 +133,13 @@ itk::Image<tissues_size_t, 2>::Pointer SliceHandlerItkWrapper::GetTissuesSlice()
 	return _GetITKView2D(all_slices.at(_handler->active_slice()), dims, spacing);
 }
 
+itk::ImageRegion<3> SliceHandlerItkWrapper::GetActiveRegion() const
+{
+	itk::Index<3> start = { 0, 0, _handler->start_slice() };
+	itk::Size<3> size = { _handler->width(), _handler->height(), static_cast<size_t>(_handler->end_slice() - _handler->start_slice()) };
```

Comment:
_handler->end_slice() - _handler->start_slice() can never be < 0?? Is that coming from some user interaction?